src/Module/Api/Twitter/Statuses/Update.php

```diff
@@ -153,9 +153,9 @@ public function post(array $request = [])
 					continue;
 				}
 
-				Photo::setPermissionForRessource($media[0]['resource-id'], $uid, $item['allow_cid'], $item['allow_gid'], $item['deny_cid'], $item['deny_gid']);
+				Photo::setPermissionForResource($media[0]['resource-id'], $uid, $item['allow_cid'], $item['allow_gid'], $item['deny_cid'], $item['deny_gid']);
 
-				$ressources[] = $media[0]['resource-id'];
+				$resources[] = $media[0]['resource-id'];
```

Comment:
I did look for column aligned tables, but I guess I missed column aligned source-code...